pkg/dns/ibm/private/dnssvcs_provider.go

```diff
@@ -0,0 +1,259 @@
+package private
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/networking-go-sdk/dnssvcsv1"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+	common "github.com/openshift/cluster-ingress-operator/pkg/dns/ibm"
+	dnsclient "github.com/openshift/cluster-ingress-operator/pkg/dns/ibm/private/client"
+	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	iov1 "github.com/openshift/api/operatoringress/v1"
+)
+
+var (
+	_   dns.Provider = &Provider{}
+	log              = logf.Logger.WithName("dns")
+
+	// validTTLs is a list of TTLs that are permitted by IBM Cloud DNS Services.
+	validTTLs = sets.NewInt64(1, 60, 120, 300, 600, 900, 1800, 3600, 7200, 18000, 43200)
+
+	// defaultDNSSVCSRecordTTL is the default TTL used when a DNS record
+	// does not specify a valid TTL.
+	defaultDNSSVCSRecordTTL = int64(120)
```

Comment:
`defaultDNSSVCSRecordTTL` can be declared as a const.

Reply:
Changed it to const.